acpi/src/fadt.rs

```diff
@@ -127,4 +127,31 @@ impl Fadt {
             other => PowerProfile::Reserved(other),
         }
     }
+
+    pub fn pm_timer_block(&self) -> Result<Option<GenericAddress>, AcpiError> {
+        let raw = unsafe {
+            self.x_pm_timer_block.access(self.header().revision).or_else(|| {
+                if self.pm_timer_block != 0 {
+                    Some(RawGenericAddress {
+                        address_space: 0,
```

Comment:
If the older `pm_timer_block` is in I/O space (I think it is, as the spec says "system port address", but I wish that was clearer), shouldn't this be `1`?